--- DBD-mysql-4.050/dbdimp.h~	2019-01-08 20:26:37.000000000 +0100
+++ DBD-mysql-4.050/dbdimp.h	2019-02-21 09:06:19.065840598 +0100
@@ -413,6 +413,9 @@ extern MYSQL* mysql_dr_connect(SV*, MYSQ
 
 extern int mysql_db_reconnect(SV*);
 int mysql_st_free_result_sets (SV * sth, imp_sth_t * imp_sth);
+#if MYSQL_VERSION_ID >= MULTIPLE_RESULT_SET_VERSION
+int dbd_st_more_results(SV* sth, imp_sth_t* imp_sth);
+#endif
 int mysql_db_async_result(SV* h, MYSQL_RES** resp);
 int mysql_db_async_ready(SV* h);
 
