--- DBD-mysql-4.006/dbdimp.c~	2007-12-11 21:41:50.000000000 +0000
+++ DBD-mysql-4.006/dbdimp.c	2008-03-02 16:18:30.667511220 +0000
@@ -2760,7 +2760,7 @@
 
       imp_sth->done_desc = 0;
     }
-    (imp_dbh->mysql).net.last_errno= 0;
+/*  (imp_dbh->mysql).net.last_errno= 0;  */
     return 1;
   }
 }
@@ -3362,7 +3362,7 @@
   }
 
   /* fix from 2.9008 */
-  (imp_dbh->mysql).net.last_errno = 0;
+/*(imp_dbh->mysql).net.last_errno = 0;  */
 
 #if MYSQL_VERSION_ID >=SERVER_PREPARE_VERSION
   if (imp_sth->use_server_side_prepare)
